### PR TITLE
Fixing typos, styles, and updating images

### PR DESCRIPTION
**Pull request by johngschulz**
There is a broken link to the vector tiles page that needs attention still.

### CODE REVIEW

File: docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.. _cartography.mbstyle.tutorial.polygon:
+Padauk.. _cartography.mbstyle.tutorial.polygon:
```
Comment (tbarsballe) [inline docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst]: This looks wrong here

File: docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst [hunk 2]
```diff
@@ -209,24 +209,24 @@ Labels can be applied to any layer using a `symbol layer <https://www.mapbox.com
...
       * - ``"text-transform": "uppercase"``
         - Change the label text to uppercase
-      * - ``text-size: 14``
+      * - ``"text-size": 14``
         - Change the font size to 14
-      * - ``"text-font": ["PT Sans"]``
-        - Change the font to PT Sans
-      * - ``text-max-width: 100``
+      * - ``"text-font": ["Padauk"]``
+        - Change the font to Padauk
```
Comment (tbarsballe) [inline docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst]: Is Padauk a Serif or Sans-Serif font? We are trying to use Serif for city names and Sans Serif for country names.

Comment (johngschulz) [reply docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst]: Ah, my apologies it is Sans - I'll look for a Serif alternative for the cities.

File: docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst (hunk 1 above)
Comment (johngschulz) [reply docs/usermanual/source/cartography/mbstyle/tutorial/polygon.rst]: Fixed and pushed

**Comment by tbarsballe** [comment]
Missing the "capitols" -> "capitals" change in `map.rst` and `sprites.json`.

Everything else looks good.

**Comment by johngschulz** [comment]
Fixed, I only found 'capitol' in the map.rst and appeared to have already been updated in the sprite.json file.

**Pull request merged**